rules.py

```diff
@@ -396,19 +403,22 @@ def _verify_data(self, file_path):
 
     def _get_ocd_data(self):
         """Checks if OCD file is in ~/cache, downloads it if not."""
-        cache_directory = os.path.expanduser(self.CACHE_DIR)
-        countries_file = "{0}/{1}".format(cache_directory, self.github_file)
-        if not os.path.exists(countries_file):
-            if not os.path.exists(cache_directory):
-                os.makedirs(cache_directory)
-            self._download_data(countries_file)
+        if self.local_file:
+            countries_file = self.local_file
         else:
-            if self.check_github:
-                last_mod_date = datetime.fromtimestamp(
-                    os.path.getmtime(countries_file))
-                latest_github_commit_date = self._get_latest_commit_date()
-                if last_mod_date < latest_github_commit_date:
-                    self._download_data(countries_file)
+            cache_directory = os.path.expanduser(self.CACHE_DIR)
```

Comment:
Take what's currently the docstring and move it here, and then update the docstring to apply to both the local file and downloaded file case.